### PR TITLE
Mobile styling

### PR DESCRIPTION
**Pull request by BereketRetta**
This is the mobile version for the portfolio website

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -84,5 +84,174 @@ <h1>
...
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul>
+            <button class="popup">See Project</button>
+          </div>
+        </div>
+        <div class="item">
+          <div class="placeholder"></div>
```
Comment (HENRYKC24) [inline index.html:108]: I will not fail to commend you for making sure you stick to the project requirement and adherence to quality code :+1: 
You nailed this part :dart: Your code is clean:ok_hand:
:clap: :clap: :clap:

File: mobile.css [hunk 2]
```diff
@@ -137,3 +145,314 @@ p {
...
+
+.placeholder {
+  background: url(./images/Screenshot24.png) no-repeat center center/cover;
+  height: 50%;
+  margin: auto;
+}
+
+.text {
+  display: flex;
+  flex-direction: column;
```
Comment (HENRYKC24) [inline mobile.css:183]: You also made use of the `grid` and `flex` CSS properties to style the elements as required. Good job:+1:

**Review by HENRYKC24 — CHANGES_REQUESTED** [review]
## Hi @BereketRetta :wave:, 

### Good job so far:+1:
around the corner

## Highlight!
- Close to perfect mobile version ✔️
- Enormous effort exerted ✔️
- Clean code ✔️
- Superb CSS ✔️

# Required Changes ♻️ ⛔️
While you made a frantic effort in this project, there are some issues that you still need to address.!
- [x] You need to document our project professionally. Missing in your README.md file are :point_down:
1. The description of your readme file only reads "This is a simple hello world project". 
image
2. Please, we can enumerate the processes for running the app locally. This is a good documentation approach and it will give you credit when a recruiter views your project.
image
3. Having a link to the issue page is very necessary for your readme file. Please, let's try to add one.
image
- [x] I also noticed that your LinkedIn link is not working. Please, check what is breaking the link.
- [x] There is apparently a missing MIT Licence file in your project. Please add it.
image
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by carlos-ssh** [comment]
# 🚫⚠️🛑 INVALID SUBMISSION 🚫⚠️🛑
Hi @BereketRetta , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is missing the features requested by the last previews. Kindly improve your `README` file and request another code review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by BereketRetta** [comment]
@carlos-ssh I have improved all the readme requirements just now can you re-check it for me, please

**Comment by ShoiraTa** [comment]
Hi @BereketRetta , I am reviewing your code, but I cannot see improved `readme`, maybe you did not push the changes ?

**Comment by BereketRetta** [comment]
dear @ShoiraTa, I have pushed the changes onto main

**Comment by BereketRetta** [comment]
Screen Shot 2021-12-04 at 6 56 23 AM

**Comment by BereketRetta** [comment]
@ShoiraTa should I submit another review

**Comment by ShoiraTa** [comment]
I have seen now, please change the name of the project its still hello world 😁, and please next time do not push chnages directly to main, all files should be pushed after review 😉

**Comment by ShoiraTa** [comment]
### **STATUS: APPROVED  ✔️** 
------
Hi @BereketRetta ,  

Your project is complete!  Congratulations!  🎉💯🌟 



Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ShoiraTa** [comment]
No need to submit for another review, please check your dashboard, I have approved it  👍🏻

**Comment by BereketRetta** [comment]
Thank you!

**Comment by carlos-ssh** [comment]
I checked up again and still missing the readme update. 

- Kindly check the readme file from this branch and you will see that this readme contains information from another project.

**Comment by carlos-ssh** [comment]
# 🚫⚠️🛑 INVALID SUBMISSION 🚫⚠️🛑
Hi @BereketRetta , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that your Pull Request it's approved. Kindly open a new pull request.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**